### PR TITLE
Removed procedure file for 3.1

### PR DESCRIPTION
**Pull request by bangelic**
It was verified that we are going to postpone upgrade with LEAPP for the time being and publish it later in the Z stream (6.11.1). It was requested that the file "proc_upgrading-project-from-el7-toel8.adoc" be removed from the document structure and also removed from the master.adoc for 3.1.

Cherry-pick into:

* [ ] Foreman 3.3
* [ ] Foreman 3.2
* [ ] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Review by maximiliankolb — APPROVED** [review]
The change LGTM. Can someone else from "we" confirm?

**Review by evgeni — CHANGES_REQUESTED** [review]
I can not, especially not for Foreman/Katello which we just enabled yesterday in https://github.com/theforeman/foreman-documentation/pull/1341

File: guides/common/modules/proc_upgrading-project-from-el7-to-el8.adoc [hunk 1]
```diff
@@ -1,205 +0,0 @@
-[id="Upgrading-Project-from-Enterprise-Linux-7-to-Enterprise-Linux-8_{context}"]
```
Comment (evgeni) [inline guides/common/modules/proc_upgrading-project-from-el7-to-el8.adoc:1]: please don't delete that file

**Comment by evgeni** [comment]
We're going to include that after all, closing.

**Pull request closed without merging**